utils/user_info.py

```diff
@@ -38,7 +39,7 @@ def has_c_ownership(curr_user, curr_contest):
     except Contest.DoesNotExist:
         logger.warning('Contest id %ld does not exsit!' % curr_contest.id)
 
-    ownership = (curr_user.username == curr_contest.owner)
+    ownership = (curr_user == curr_contest.owner)
```

Comment:
curr_user is an user object, curr_contest.owner is an user object, too.
In Bruce's opinion, you can change it to curr_user.username == curr_contest.owner.username